using-mobile-extensions/adobe-journey-optimizer/README.md

```diff
@@ -80,164 +87,199 @@ Follow these steps to integrate the Adobe Journey Optimizer extension.
 
 2. Import the Mobile Core, Edge, Edge Identity, and Messaging libraries:
 
-### Swift
+   #### Swift
 
-```swift
-// AppDelegate.swift
-import AEPCore
-import AEPEdge
-import AEPEdgeIdentity
-import AEPMessaging
-```
+    ```swift
+    // AppDelegate.swift
 
-### Objective-C
+    import AEPCore
+    import AEPEdge
+    import AEPEdgeIdentity
+    import AEPMessaging
+    ```
+
+    #### Objective-C
+
+    ```objc
+    // AppDelegate.h
+
+    @import AEPCore;
+    @import AEPEdge;
+    @import AEPEdgeIdentity;
+    @import AEPMessaging;
+    ```
+
+{% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+
+This extension is built on the AEPCore (3.x) and it is not compatible with ACPCore (2.x). Please follow [the guide for migrating to the Swift AEPCore](https://aep-sdks.gitbook.io/docs/resources/migrate-to-swift).
```

Comment:
nit: update this link to use a relative path to the migration docs (shout out to @swarna04 😄 )